setup.cfg

```diff
@@ -92,4 +93,5 @@ addopts = --doctest-rst
 
 [flake8]
 exclude = extern
-select = E101 W191 W291 W292 W293 W391 E11 E502 E722 E901 E902
+select = F, W, E101 W191 W291 W292 W293 W391 E11 E502 E722 E901 E902
```

Comment:
Can we can remove the W### from this list?  Seems like they should already be selected by `F`

Reply:
Done